.github/workflows/maven.yml

```diff
@@ -16,18 +16,13 @@ jobs:
     timeout-minutes: 10
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up JDK 11
-      uses: actions/setup-java@v2
+      uses: actions/setup-java@v3
       with:
         java-version: '11'
-        distribution: 'adopt'
-    - name: Cache Maven packages
-      uses: actions/cache@v2
```

Comment:
Does the cache work in the distribution? It seems the build is downloading the maven packages again.

Reply:
```
Creating settings.xml with server-id: github
Writing to /home/runner/.m2/settings.xml
Received 180355072 of 201222982 (89.6%), 171.8 MBs/sec
Received 201222982 of 201222982 (100.0%), 155.5 MBs/sec
Cache Size: ~192 MB (201222982 B)
/usr/bin/tar --use-compress-program zstd -d -xf /home/runner/work/_temp/3792ca9a-63ea-4a15-9abc-64b[34](https://github.com/slackhq/kaldb/actions/runs/3183901636/jobs/5192666231#step:3:36)fd7c3ef/cache.tzst -P -C /home/runner/work/kaldb/kaldb
Cache restored successfully

Cache restored from key: setup-java-Linux-maven-2[35](https://github.com/slackhq/kaldb/actions/runs/3183901636/jobs/5192666231#step:3:37)149182e131ef7caef7549b08a[37](https://github.com/slackhq/kaldb/actions/runs/3183901636/jobs/5192666231#step:3:39)7fce4d58d3[39](https://github.com/slackhq/kaldb/actions/runs/3183901636/jobs/5192666231#step:3:41)948b1bbafb7b4cab7fc492
```

I see this in the output in the `Setup with JDK 11 ` action